scripts/biomedical/Human-GEM_Human1/scripts/format_genes.py

```diff
@@ -0,0 +1,57 @@
+"""
+This script formats gene.tsv and geneRoles.tsv files to
+combine with tMCF files for import to data commons
+"""
+import sys
+import os
+import numpy as np
+import pandas as pd
+
+
+def main():
```

Comment:
too much is stored in main. This should be broken up into several functions. General good coding practices is that functions should only be max 20 lines and that main() should be a wrapper function. Break up your code into smaller functions and call these from wrapper functions.